### ISSUE TITLE
Customizable Comment widget

### ISSUE THREAD

**Issue by Wissenswerkstatt** [pojome/elementor#6110]
## Prerequisites

- [ x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x ] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
There are no ways to customize the comments. That's quite frustrating. 

**Describe the solution you'd like**
Please add some customization possibilities to the comments widget. It'd be nice to rearrange the comment fields, to have the opportunity to change the fontsize and so on.

**Comment by Wissenswerkstatt** [comment]
Is nobody else missing the option to modify the comment section?

**Comment by TomasConnor** [comment]
±1

**Comment by artconic** [comment]
Not only font size. Add.. Most importantly.... a script to import from LinkedIn, pinterest, etc. Able 2 add title, company, year, company logo.

**Comment by Cecile-Lebleu** [comment]
This is vital for a good blog design!! Selecting the size of the comment box, padding and margin of each element, typography, colors, borders, the usual!
Please, please implement it soon. I can't even find a plugin or add-on to customize this, and installing a whole theme that will slow everything down just to customize this doesn't make sense!

**Comment by Roadspill** [comment]
I second this feature.

Using Elementor Hello Theme makes the comments section look horrendous, and with no way to customize via Elementor almost makes the Hello Theme useless if it doesn't look good, and we can't change it.

**Comment by Roadspill** [comment]
Yes I know I can edit and customize the code, but isn't the premise of a page builder to be able to edit fully within the builder?

**Comment by mareklexuan** [comment]
I second this proposal. I am also using the Hello Elementor theme and redesigning the comments section manually is just super annoying, considering it is a basic feature for every blog.

I put together some ideas (my common issues with comments section) for the features this widget can have, just to push it a little bit.
 
### Comments

**Title**
- Custom content/text
- Typography settings

**Comments**
- Enable/disable avatar
- Custom format of the meta ("XY says", date format)
- "Reply" - custom text, option to make it a button and not only text
- Typography for each part (meta, content, reply)
- Design options for the "reply" button

### Comments form

**Title**
- Custom content/text
- Typography settings

**Comment Notes**
- Custom content/text
- Typography settings

**Comments form fields**
- Choose which fields to display
- In each field, choose which is required
- Display/hide field title
- Typography settings for the field titles
- Field title position (above, left)
- Typography settings for the field required icon (icon, size, color)

**Submit**
- Add tick box for "agree with processing personal information" (custom text, with a link)
- Customizable submit button (text, the typography of the text, design of the button, position)
- Enable/disable cookie feature ("Save my name, email, and website in this browser for the next time I comment.")
- Ability to subscribe to replies

### General

- Preview of design (when creating in Elementor, I am signed in but I have maintenance mode on, so I can´t preview the design of comment section)
- All those usuals like borders, background colors, space between form fields, etc.

**Comment by 79ho3ein** [comment]
+1
This feature is very, very, very important. I hope it will be added as soon as possible.

**Comment by saawood13** [comment]
+1 I am appalled that this is not already an advanced widget?
